index.html

```diff
@@ -0,0 +1,30 @@
+<!--HTML layout page-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body class="main-content">
+    <header class="mainheader">
+        <h2 class="usernamehandle">bettercallsully</h2>

```

Comment:
- [ ] Kindly notice that the logo of the page is the main navigator link on the website. So, it'd be way too better if it has a link `<a>` tag. Let's work on that!